packages/core/stories/patterns/contact-details/contact-details.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  StackLayout,
+  Avatar,
+  Text,
+  Card,
+  Button,
+  Tooltip,
+  ResponsiveProp,
+  LayoutDirection,
+} from "@salt-ds/core";
+import {
+  WomanIcon,
+  HomeIcon,
+  MessageIcon,
+  LocationIcon,
+  CallIcon,
+  ChatIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+import { useState } from "react";
+
+export default {
+  title: "Patterns/Contact Details",
+} as Meta;
+
+const basicContact = {
+  primary: "Alex Brailescu",
+  secondary: "Blackrock Advisors (UK) Limited",
+  metadata: [
+    {
+      label: "Role",
+      value: "Analyst",
+      icon: <WomanIcon />,
+    },
+    {
+      label: "Location",
+      value: "London, GBR",
+      icon: <LocationIcon />,
+    },
+    {
+      label: "Office",
+      value: "+44 2077 431102",
+      icon: <HomeIcon />,
+    },
+    {
+      label: "Email",
+      value: "alex.brailescu@blackrock.com",
+      icon: <MessageIcon />,
+    },
+  ],
+};
+
+export interface ContactDetailsProps {
+  avatarSize?: number;
+  gap?: number;
+  direction?: ResponsiveProp<LayoutDirection>;
+  avatarSrc?: string;
+}
+
+export const ContactDetails = ({
+  avatarSize = 2,
+  gap = 1,
+  direction = "column",
+  avatarSrc = "https://storybook.saltdesignsystem.com/assets/avatar-840503bf.png",
```

Comment:
This doesn't seem like a safe URL to rely on?

Reply:
That is included via an import so that during the build process when the image is processed and a hash is added to the end, the URL is always correct. This relies on that avatar png hash being constant and never changing.

Reply:
What would you suggest as an alternative to using the URL or is there a catch all so that the image can be used irrespective of the hash at the end ?

Reply:
https://github.com/jpmorganchase/salt-ds/blob/bf135621ace98e6f787b21ba6ac6d75ec3e9f204/packages/core/stories/avatar/avatar.stories.tsx#L11

Reply:
https://github.com/jpmorganchase/salt-ds/blob/bf135621ace98e6f787b21ba6ac6d75ec3e9f204/packages/core/stories/avatar/avatar.stories.tsx#L41

Reply:
updated - ty